### PR TITLE
SAK-47964 Groups manager and date manager are not taking properly site locale

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thanks Jose!

File: site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/tool/MainController.java [hunk 1]
```diff
@@ -63,11 +64,13 @@ public class MainController {
...
     private PreferencesService preferencesService;
 
+    @Autowired
+    private static final ResourceLoader rb = new ResourceLoader("Messages");
+
     @GetMapping(value = {"/", "/index"})
     public String showIndex(@RequestParam(required=false) String code, Model model, HttpServletRequest request, HttpServletResponse response) {
 
-        String userId = sessionManager.getCurrentSessionUserId();
-        final Locale loc = StringUtils.isNotBlank(userId) ? preferencesService.getLocale(userId) : Locale.getDefault();
```
Comment (mpellicer) [inline site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/tool/MainController.java:70]: Do you know if "private PreferencesService preferencesService;" can be removed and cleanup?

File: site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java [hunk 2]
```diff
@@ -68,8 +68,7 @@ public class MainController {
     public String showIndex(Model model, HttpServletRequest request, HttpServletResponse response) {
         log.debug("showIndex()");
         
-        String userId = sessionManager.getCurrentSessionUserId();
-        final Locale locale = StringUtils.isNotBlank(userId) ? preferencesService.getLocale(userId) : Locale.getDefault();
```
Comment (mpellicer) [inline site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java:72]: Do you know if "private PreferencesService preferencesService;" can be removed and cleanup?

Comment (josecebe) [reply site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java:72]: Yes, in both cases those variables could have been removed, so I did it. Thanks!

Comment (mpellicer) [reply site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java:72]: Nice cleanup!

**Pull request merged**